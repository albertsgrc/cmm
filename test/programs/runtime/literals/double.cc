/*
    description{Tests double literal parsing}
*/

#include <iostream>
using namespace std;

void print(double c) {
    cout << c << endl;
}

int main() {
    print(.5);
    print(0.34246238);
    print(55223.23728);
    print(.005);
    print(00.244402);
    print(1.2242);
}