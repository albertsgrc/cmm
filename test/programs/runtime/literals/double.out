0.5
0.342462
55223.2
0.005
0.244402
1.2242
